src/pages/Recipes.js

```diff
@@ -9,6 +9,8 @@ import Col from 'react-bootstrap/Col';
 import Container from 'react-bootstrap/Container';
 import { Button } from 'react-bootstrap';
 import Card from 'react-bootstrap/Card';
+import Dropdown from 'react-bootstrap/Dropdown';
```

Comment:
You can turn this into `import { Button, Card, Dropdown } from 'react-bootstrap'`

Reply:
That way there is no need for line 11 and 12